modules/core/src/v2/wallet.ts

```diff
@@ -122,9 +122,8 @@ export interface PrebuildTransactionResult extends TransactionPrebuild {
 export interface WalletSignTransactionOptions {
     txPrebuild?: TransactionPrebuild;
     prv?: string;
-    userKeychain?: Keychain,
-    backupKeychain?: Keychain,
-    bitgoKeychain?: Keychain,
+    pubs?: string[];
```

Comment:
This is a non-backwards change to the API. Could you please properly indicate that in the commit message according to the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-both--and-breaking-change-footer) rules? This is important when I go to publish since we need to make this a major version bump (which we're planning on doing shortly anyway, since we need to drop node 12 support soon).

Even better would be to avoid a breaking change here, but I think it's a good change and the timing is not bad either.

Reply:
I actually don't think this is a breaking change here: if `pubs` is empty, the keys are fetched in `signTransaction`

Reply:
However `AbstractUtxoCoin.prototype.signTransaction` now requires `pubs` so I guess that is a breaking change. I will explain this in the commit message.

Reply:
the removal of `userKeychain`/`backupKeychain`/`bitgoKeychain` from the interface is a breaking change I suppose

Reply:
Timing is good because these are relatively new - I believe I added these within the past few weeks for taproot purposes.